package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "build": "npm run clean && npm run check-style && node_modules/webpack/bin/webpack.js",
     "build-style": "cd node_modules/cloudgov-style && npm install && npm run build",
-    "check-style": "/bin/bash -c '[[ $(npm ls cloudgov-style) =~ \"github.com\" ]] && npm run build-style || exit 0'",
+    "check-style": "/bin/bash -c '[[ $(npm ls cloudgov-style) =~ \"git://github.com\" ]] && npm run build-style || exit 0'",
```

Comment:
Hmm, I think I usually see `git+https://` in `package.json`. It's also uncommon, but folks might also put an ssh uri too. What does adding the protocol solve?